### PR TITLE
Add post about in-person meeting

### PR DESCRIPTION
**Pull request by rgaiacs**
@swcarpentry/board Could you review?

### CODE REVIEW

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+In the following weeks we will wrote more about the results of the in-person
+meeting on the blog but for now we want to give you an overview.
+
+## Partnerships
+
+Our Executive Director is doing an amazing job increasing the number of Software
+Carpentry [partners](http://software-carpentry.org/scf/partners/). Last year,
+we were asked to provide a join partnership agreement with [Data
+Carpentry](http://datacarpentry.org/)
+and after months of work this agreement are now
```
Comment (k8hertweck) [inline _posts/2016/05/2016-05-11-first-tertile.md]: "this agreement is"

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Instructor Training
+
+Last year we tried a lot of models for Instructor Training
+that now are more stable and has an [write
+version](http://swcarpentry.github.io/instructor-training/)
+thanks to the work of our Instructor Trainers.
+
+We will continue improving our Instructor Training programme
+and increasing our capacity of delivery it to attend the high demand.
```
Comment (k8hertweck) [inline _posts/2016/05/2016-05-11-first-tertile.md]: increasing our capacity to meet high demand.

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Communication is our greatest weakness and we are going to work really hard to
+improve it. (**This post is part of this goal.**) Something that we decided
+during the in-person meeting was provide [an official space on
+Facebook](https://www.facebook.com/groups/17942228448/) because on some
+countries and some audiences this is the first place that they will look for us.
+
+## Diversity
+
+We will continue to support an diverse community and health spaces for
```
Comment (k8hertweck) [inline _posts/2016/05/2016-05-11-first-tertile.md]: a diverse community and healthy spaces

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+communication. We have a [Code of
+Conduct](http://software-carpentry.org/conduct) to help with that.
+
+## Future Directions
+
+We will continue to work close with others communities that are aligned with our
```
Comment (k8hertweck) [inline _posts/2016/05/2016-05-11-first-tertile.md]: work closely with other communities

**Comment by k8hertweck** [comment]
Great post, thanks for summarizing everything so succinctly! I added a few minor wording suggestions.

File: _posts/2016/05/2016-05-11-first-tertile.md (hunk 1 above)
Comment (weaverbel) [inline _posts/2016/05/2016-05-11-first-tertile.md]: "In the following weeks we will write more"

File: _posts/2016/05/2016-05-11-first-tertile.md (hunk 4 above)
Comment (weaverbel) [inline _posts/2016/05/2016-05-11-first-tertile.md]: Please add:
"We will also create an Instagram account to display and share photographs from Software Carpentry workshops and activities."

**Comment by weaverbel** [comment]
Thanks @rgaiacs - your poor fingers must be exhausted after all that minute-taking during the weekend. Thanks for doing it and for writing this post too. <3 <3

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "First bimonthly report from 2016 Steering Committee"
+date: 2016-05-11
+time: "10:00:00"
+category: ["Community"]
+---
+On May 8th and 9th the Steering Committee had an in-person meeting at Cold
+Spring Harbour Laboratory to conclude the discussions that occurred on the first
```
Comment (karinlag) [inline _posts/2016/05/2016-05-11-first-tertile.md]: during the first two months

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+two months and make plans for the next ten months of activities.
+In the following weeks we will wrote more about the results of the in-person
```
Comment (karinlag) [inline _posts/2016/05/2016-05-11-first-tertile.md]: we will write

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+Our Executive Director is doing an amazing job increasing the number of Software
+Carpentry [partners](http://software-carpentry.org/scf/partners/). Last year,
+we were asked to provide a join partnership agreement with [Data
+Carpentry](http://datacarpentry.org/)
+and after months of work this agreement are now
+[available](http://software-carpentry.org/scf/join/).
+
+## Instructor Training
+
+Last year we tried a lot of models for Instructor Training
```
Comment (karinlag) [inline _posts/2016/05/2016-05-11-first-tertile.md]: we tried several models for

**Comment by jduckles** [comment]
@rgaiacs let me know when it is ready to merge.

**Comment by rgaiacs** [comment]
> let me know when it is ready to merge.

@jduckles It is ready now.

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+Spring Harbour Laboratory to conclude the discussions during the first
+two months and make plans for the next ten months of activities.
+In the following weeks we will write more about the results of the in-person
+meeting on the blog but for now we want to give you an overview.
+
+## Partnerships
+
+Our Executive Director is doing an amazing job increasing the number of Software
+Carpentry [partners](http://software-carpentry.org/scf/partners/). Last year,
+we were asked to provide a join partnership agreement with [Data
```
Comment (wking) [inline _posts/2016/05/2016-05-11-first-tertile.md:19]: “join” → “joint”?

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+Carpentry](http://datacarpentry.org/)
+and after months of work this agreement is now
+[available](http://software-carpentry.org/scf/join/).
+
+## Instructor Training
+
+Last year we tried several models for Instructor Training
+that now are more stable and has an [write
+version](http://swcarpentry.github.io/instructor-training/)
```
Comment (wking) [inline _posts/2016/05/2016-05-11-first-tertile.md:28]: “write version” → “written version”.  And maybe kick off a new sentence: “… Training.  Now the model has stabilized and has a written version, thanks to …”.

File: _posts/2016/05/2016-05-11-first-tertile.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+## Diversity
+
+We will continue to support a diverse community and healthy spaces for
+communication. We have a [Code of
+Conduct](http://software-carpentry.org/conduct) to help with that.
+
+## Future Directions
+
+We will continue to work close with other communities that are aligned with our
```
Comment (wking) [inline _posts/2016/05/2016-05-11-first-tertile.md:60]: “close” → “closely”.

**Pull request merged**